qiskit/_qiskiterror.py

```diff
@@ -6,12 +6,12 @@
 # the LICENSE.txt file in the root directory of this source tree.
 
 """
-Exception for errors raised by the QISKit SDK.
+Exception for errors raised by the Qiskit.
 """
 
 
-class QISKitError(Exception):
-    """Base class for errors raised by the QISKit SDK."""
+class QiskitError(Exception):
```

Comment:
So this is a pretty big breaking change. This is our main exception class which means that it's almost certainly being caught by user programs which will start not catching once we make this change. Basically any place we raise an exception it's either `QISKitError` or a subclass of it. I'm not sure the benefit here of consistent capitalization is worth changing something so core. If it is we definitely have to deprecate it or alternatively make `QiskitError` a subclass of `QISKitError` so that people's code catching `QISKitErrror` won't break if we change to `QiskitError` everywhere else in the code base. (hopefully that makes sense, it's hard with it being the same word with different capitalization :) )

Reply:
ok, i let you put something together. You don't think it is bad to keep an old name around.